src/services/stripe.service.ts

```
@@ -0,0 +1,265 @@
+   * Handle checkout session completed
+   */
+  private async handleCheckoutSessionCompleted(session: any) {
+    const { userId, planId } = session.metadata;
+    // Update user subscription status
+    await prisma.user.update({
+      where: { id: userId },
+      data: { is_subscription_active: true }
+    });
+
+    // Guardar el Stripe Customer ID en el usuario
+    const customerId = session.customer || (session.subscription && session.subscription.customer);
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)